apps/daimo-mobile/src/view/screen/send/SendSuccessScreen.tsx

```diff
@@ -0,0 +1,76 @@
+import { fullDateAndTime } from "@daimo/common";
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+import { useCallback, useEffect, useRef } from "react";
+import { View } from "react-native";
+
+import { AmountChooser } from "../../shared/AmountInput";
+import { AnimatedCheck, AnimatedCheckRef } from "../../shared/AnimatedCheck";
+import { ButtonBig } from "../../shared/Button";
+import { ScreenHeader } from "../../shared/ScreenHeader";
+import Spacer from "../../shared/Spacer";
+import { ParamListSend, navResetToHome, useNav } from "../../shared/nav";
+import { ss } from "../../shared/style";
+import { TextBody, TextCenter, TextLight } from "../../shared/text";
+
+type Props = NativeStackScreenProps<ParamListSend, "SendSuccess">;
+
+export function SendSuccessScreen({ route }: Props) {
+  const { dollars, recipient } = route.params || {};
+  const nav = useNav();
+  const nDollars = parseFloat(dollars || "");
+  const checkRef = useRef<AnimatedCheckRef>(null);
+
+  useEffect(() => {
+    checkRef?.current?.play();
+  }, []);
+
+  const onExit = () => navResetToHome(nav);
+  const onBack = () =>
+    nav.navigate("SendTab", {
+      screen: "SendNav",
+      params: { autoFocus: true },
+    });
+
+  const noop = useCallback(() => {}, []);
+
+  const nowS = useRef(Date.now() / 1000).current;
+  const { day, time } = fullDateAndTime(nowS);
```

Comment:
changed this to always use the current time (`lastSendTime` could be a week ago)